map2loop/deformation_history.py

```diff
@@ -29,14 +31,15 @@ class DeformationHistory:
 
     """
 
-    def __init__(self):
+    def __init__(self, project):
         """
         The initialiser for the deformation history. All attributes are defaulted
         """
-        self.minimum_fault_length_to_export = 500.0
         self.history = []
         self.fault_fault_relationships = []
-
+        self.minimum_fault_length = project.get_minimum_fault_length()
```

Comment:
@AngRodrigues this still won't work. If the minimum fault length is set using the calculate_minimum_fault_length function, as in line 277 the minimum fault length will not be up to date with the project/config minimum fault length. 

The project needs to be set as an attribute of the deformation_history class that is referenced back to when updating the value either using a property accessor or directly using the project.set_minimum_fault_length.

Reply:
It also calls get_minimum_fault_length() before reading the config file because its set when the deformationhistory object is created.